.changeset/cold-ties-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/button": patch
```

Comment:
No need for a changeset (well rather, make this empty) since it's just fixing a story, therefore nothing to release :)

Thanks for the fix!

Reply:
Thanks @HeartSquared, that's done now.